lib/rest-adapter.js

```diff
@@ -425,6 +433,11 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  // if (fastRouter.canRegister(route, handler)) {
```

Comment:
I don't think it's reasonable to expect that fastRouter will support all different route flavours supported by express (e.g. regular expressions). That's why it's important to call `canRegister` (or perhaps `canHandle` is a better name?) here.